README.md

```diff
@@ -94,3 +129,385 @@ async def callback_handler(request: Request) -> JSONResponse:
         status_code=HTTPStatus.ACCEPTED,
     )
 ```
+
+## Примеры
+
+
+### Получение сообщений
+
+*([подробное описание функции](
+https://ccsteam.atlassian.net/wiki/spaces/SMARTAPP/pages/311001185/Bot+API+BotX+API#%D0%9F%D0%BE%D0%BB%D1%83%D1%87%D0%B5%D0%BD%D0%B8%D0%B5-%D1%81%D0%BE%D0%BE%D0%B1%D1%89%D0%B5%D0%BD%D0%B8%D0%B9/%D1%81%D0%B8%D1%81%D1%82%D0%B5%D0%BC%D0%BD%D1%8B%D1%85-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9))*
+
+```python
+from uuid import UUID
+
+from pybotx import *
+
+ADMIN_HUIDS = (UUID("123e4567-e89b-12d3-a456-426614174000"),)
+
+collector = HandlerCollector()
+
+
+@collector.command("/visible", description="Visible command")
+async def visible_handler(_: IncomingMessage, bot: Bot) -> None:
+    # Обработчик команды бота. Команда видимая, поэтому описание
+    # является обязательным.
+    print("Hello from `/visible` handler")
+
+
+@collector.command("/_invisible", visible=False)
+async def invisible_handler(_: IncomingMessage, bot: Bot) -> None:
+    # Невидимая команда - не отображается в списке команд бота
+    # и не нуждается в описании.
+    print("Hello from `/invisible` handler")
+
+
+async def is_admin(status_recipient: StatusRecipient, bot: Bot) -> bool:
+    return status_recipient.huid in ADMIN_HUIDS
+
+
+@collector.command("/admin-command", visible=is_admin)
+async def admin_command_handler(_: IncomingMessage, bot: Bot) -> None:
+    # Команда показывается только если пользователь является админом.
+    # Список команд запрашивается при открытии чата в приложении.
+    print("Hello from `/admin-command` handler")
+
+
+@collector.default_message_handler
+async def default_handler(_: IncomingMessage, bot: Bot) -> None:
+    # Если команда не была найдена, вызывается `default_message_handler`,
+    # если он определён. Такой обработчик может быть только один.
+    print("Hello from default handler")
+```
+
+
+### Получение системных событий
+
+*([подробное описание функции](
+https://ccsteam.atlassian.net/wiki/spaces/SMARTAPP/pages/311001185/Bot+API+BotX+API#%D0%9F%D0%BE%D0%BB%D1%83%D1%87%D0%B5%D0%BD%D0%B8%D0%B5-%D1%81%D0%BE%D0%BE%D0%B1%D1%89%D0%B5%D0%BD%D0%B8%D0%B9/%D1%81%D0%B8%D1%81%D1%82%D0%B5%D0%BC%D0%BD%D1%8B%D1%85-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9))*
```

Comment:
Would you like to shorten the link?
Example: https://bit.ly/3OF0id6
I don't think they change often.

Reply:
Good idea. Although link is only in sources, I think it's overkill here.